Multisig/UI/Transactions/TransactionsList/LoadingTransactionListView.swift

```diff
@@ -27,9 +27,9 @@ struct LoadingTransactionListView: View {
                                 reload: model.reload,
                                 loadMore: model.loadMore)
         }
-        .onAppear {
-            trackEvent(.transactions)
-        }
+//        .onAppear {
+//            trackEvent(.transactions)
+//        }
```

Comment:
can be removed?

Reply:
I didn't want to change anything in the old implementation so I kept them for notice. I will replace them by TODO comment